test/db/cmd/cmd_search

```diff
@@ -1023,3 +1023,57 @@ EXPECT=<<EOF
 0x00000005 hit0_0 a5a5bf
 EOF
 RUN
+
+NAME=config search smartcase
+CMDS=<<EOF
+el~rzil
+el~+rzil
+el~RzIL
```

Comment:
maybe add a `echo` between these lines to make sure the EXPECT part is correctly split and you know which output is generate from which command.